lib/team_statistics.rb

```diff
@@ -0,0 +1,310 @@
+    end
+    total_2013 = season_win_data_set.find_all do |game|
+      game[0].start_with?("2013") && game[1] == team_id
+    end.count
+    season_win_data_set.map do |game|
+      if game[0].start_with?("2013")
+        total_season_games["20132014"] = total_2013
+      end
+    end
+    total_2014 = season_win_data_set.find_all do |game|
+      game[0].start_with?("2014") && game[1] == team_id
+    end.count
+    season_win_data_set.map do |game|
+      if game[0].start_with?("2014")
+        total_season_games["20142015"] = total_2014
+      end
+    end
+    total_2015 = season_win_data_set.find_all do |game|
+      game[0].start_with?("2015") && game[1] == team_id
+    end.count
+    season_win_data_set.map do |game|
+      if game[0].start_with?("2015")
+        total_season_games["20152016"] = total_2015
+      end
+    end
+    total_2016 = season_win_data_set.find_all do |game|
+      game[0].start_with?("2016") && game[1] == team_id
+    end.count
+    season_win_data_set.map do |game|
+      if game[0].start_with?("2016")
+        total_season_games["20162017"] = total_2016
+      end
+    end
+    total_season_games
+  end
+
+  def total_wins_by_season(team_id)
+    winning_games_by_season = {}
+    # returns for team_id '3' {"20122013"=>0, "20142015"=>0, "20162017"=>0}
+    win_results_2012 = season_win_data_set.find_all do |game|
+      game[0].start_with?("2012") && game[1] == team_id && game[2] == "WIN"
+      end.count
+      season_win_data_set.map do |game|
+        if game[0].start_with?("2012")
+            winning_games_by_season["20122013"] = win_results_2012
+        end
+      end
+    win_results_2013 = season_win_data_set.find_all do |game|
+      game[0].start_with?("2013") && game[1] == team_id && game[2] == "WIN"
+      end.count
+    season_win_data_set.map do |game|
+      if game[0].start_with?("2013")
+          winning_games_by_season["20132014"] = win_results_2013
+      end
+    end
+    win_results_2014 = season_win_data_set.find_all do |game|
+      game[0].start_with?("2014") && game[1] == team_id && game[2] == "WIN"
+      end.count
+    season_win_data_set.map do |game|
+      if game[0].start_with?("2014")
+          winning_games_by_season["20142015"] = win_results_2014
+      end
+    end
+    win_results_2015 = season_win_data_set.find_all do |game|
+      game[0].start_with?("2015") && game[1] == team_id && game[2] == "WIN"
+      end.count
+    season_win_data_set.map do |game|
+      if game[0].start_with?("2015")
+          winning_games_by_season["20152016"] = win_results_2015
+      end
+    end
+    win_results_2016 = season_win_data_set.find_all do |game|
+      game[0].start_with?("2016") && game[1] == team_id && game[2] == "WIN"
+      end.count
+    season_win_data_set.map do |game|
+      if game[0].start_with?("2016")
+          winning_games_by_season["20162017"] = win_results_2016
+      end
+    end
+    winning_games_by_season
+  end
+
+  def total_loss_by_season(team_id)
```

Comment:
When we get to refactoring, we can combine these three by passing an argument, just adding that note here.